themes/custom-theme/theme/katanox/email/messages/messages_en.properties

```diff
@@ -0,0 +1,49 @@
+emailVerificationSubject=Verify email
+emailVerificationBody=Someone has created a {2} account with this email address. If this was you, click the link below to verify your email address\n\n{0}\n\nThis link will expire within {3}.\n\nIf you didn''t create this account, just ignore this message.
```

Comment:
Do the `'` need to be doubled to be escaped? If not, let's run a quick spell check on the messages

Reply:
yep, they do. Otherwise the system will remove it